usr/share/rear/conf/default.conf

```diff
@@ -1699,7 +1699,12 @@ DUPLY_PROFILE=""
 # BACKUP=DUPLICITY
 # BACKUP_PROG="duplicity"
 #
-# DUPLICITY_PROG="/usr/bin/duplicity"
+DUPLICITY_PROG="$(which duplicity)"
```

Comment:
I am always unsure what to use either `which` or `type -P` to get the exectuable file
and if the exectuable file is really the right thing that should be called,
perhaps an alias or a function is the right thing that should be called?
For example:
<pre>
# type -a ls
ls is aliased to `_ls'
ls is /usr/bin/ls
ls is /bin/ls
ls is /usr/bin/X11/ls

# type -P ls
/usr/bin/ls

# which ls
/usr/bin/ls
</pre>
It looks as if `which` and `type -P` result the same
but I don't know if there could be subtle differences.